main.scss

```diff
@@ -1,14 +1,173 @@
+		// border-left
+		--palette-error-darkest: #{inspect(oColorsGetPaletteColor('claret-70'))};
+
+
+		// community guidelines box
+		// background
+		--palette-primary-lightest: #{inspect(oColorsGetPaletteColor('wheat'))};
+		// border
+		--palette-primary-light: #{inspect(oColorsGetPaletteColor('wheat'))};
+
+
+		// apply button in configure tab
+		--palette-success-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+		--palette-success-light: #{inspect(oColorsGetPaletteColor('teal-60'))};
+
+
+		// stream section selected tab item, selected item count badge
+		--palette-primary-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// stream section unselected tab comment count badge
+		// blockquote background
+		--palette-grey-lightest: #{inspect(oColorsGetPaletteColor('wheat'))};
+
+
+		// stream section tab bar divider
+		--palette-divider: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// respect, reply, share, report
+		--palette-grey-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// report button "reported" state
+		--palette-error-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// moderation drop down
+		// "Approved" button (approve button state)
+		--palette-success-dark: #{inspect(oColorsGetPaletteColor('black'))};
+		// "Un Feature" button (feature button state)
+		--palette-primary-dark: #{inspect(oColorsGetPaletteColor('black'))};
+	}
+
+	// Mainly for the links in the community guidelines box
+	a {
+		@include oTypographyLink;
+	}
+
+	// remove divider of the top tab bar
+	.coral-tabBar {
+		border-bottom-width: 0;
+	}
+
+	// top tab bar buttons
+	.coral-tabBar-allComments,
+	.coral-tabBar-myProfile,
+	.coral-tabBar-configure	{
+		@include oButtons;
+
+		border-width: 1px;
+
+		// normally, this should come for free with oButtons mixin
+		// but we have to declare this explicitly here because
+		// Coral styles override our styles for some reason
+		&[aria-selected=true],
+		&[aria-pressed=true],
+		&[aria-current] {
+			color: oButtonsGetColor('', 'color', 'primary') !important;
+			background-color: oButtonsGetColor('', 'background', 'primary') !important;
+			border-color: oButtonsGetColor('', 'border', 'primary') !important;
+			border-top: 1px !important;
+			border-bottom: 1px !important;
+		}
+	}
+
+	// post comment buttons
+	.coral-createComment-submit,
+	.coral-commentReply-submit,
+	.coral-createComment-signIn {
+		@include oButtons;
+
+		// normally, this should come for free with oButtons mixin
+		// but we have to declare this explicitly here because
+		// Coral styles override our styles for some reason
+		color: oButtonsGetColor('', 'color', 'primary') !important;
+		background-color: oButtonsGetColor('', 'background', 'primary') !important;
+		border-color: oButtonsGetColor('', 'border', 'primary') !important;
+	}
+
+	// spacing and divider between comments
+	.coral-comment {
+		padding-bottom: 6px;
+		padding-top: 10px;
+		border-bottom: solid 1px !important;
```

Comment:
Could these be border properties converted into shorthand?

`border-bottom: solid 1px oColorsGetPaletteColor('black-20')`

I am also not sure the `!important` is needed but I might be wrong, again lets pair to check it

Reply:
Yes, it works that way. I did the change.